development/README.md

```diff
@@ -53,21 +53,44 @@ Copy the address of the ingress `ing` and open it in a browser to display the Pr
 
 ## Configuration
```

Comment:
```suggestion
## Development
```

Reply:
Please change into Development to be compliant with the guidelines:
https://github.com/kyma-project/community/blob/master/guidelines/internal-guidelines/repository-template/template/README.md